src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexWriteOperation.cs

```diff
@@ -66,9 +66,6 @@ public override void Commit(IndexingStatsScope stats)
 
             using (Stats.ConvertStats.Start())
                 data = _converter.SetDocumentFields(key, sourceDocumentId, document, indexContext, out lowerId, _buffer.ToSpan());
-
-            if (data.IsEmpty)
-                return;
             
             using (Stats.AddStats.Start())
                 _indexWriter.Index(lowerId, data, _knownFields);
```

Comment:
@maciejaszyk Is there any way I can know here or before if the document is actually an implicit null (and/or explicit)?

Reply:
@redknightlois 
In `CoraxDocumentConverterBase` we have something in constructor:
![image](https://user-images.githubusercontent.com/86351904/168743666-266973e3-37bd-48ed-9d7c-7a058b8c7c05.png)